I/ImageView/Compat.toml

```diff
@@ -113,11 +113,12 @@ GtkObservables = "2"
 ["0.12.0"]
 Gtk4 = "0.5"
 
-["0.12.1-0"]
+["0.12.1"]
 Gtk4 = "0.6"
 
 ["0.12.2-0"]
 GtkObservables = "2.1.0-2"
+Gtk4 = "0.6.2"
```

Comment:
I think this means only 0.6.2 is allowed, is that what you want?

Reply:
Definitely not, thanks! Better now?